browser/flagr-ui/src/helpers/helpers.js

```diff
@@ -40,10 +40,27 @@ function handleErr (err) {
   }
 }
 
+function getHue(str) {
```

Comment:
do we test this somewhere?

Reply:
Apologies for my slow responses.

I can't see any front-end testing set up or any tests for the existing javascript. Quite possible I'm missing something though.
I'm happy to set up a testing framework for the front-endif there are none. In this PR or another. :)

Reply:
no worries! I'm not much aware of the FE, that's something @zhouzhuojie would know better

Reply:
Yup, frontend testing is light - may need to setup something first. I can investigate getting something simple like a small js test to start the CI workflow for frontend testing.